docs/dev/auth.rst

```diff
@@ -23,31 +29,9 @@ the SP with correct configuration.
 Configuration
 """""""""""""
 
-The following configuration entries are relevant for SAML SSO:
+Configuration of SAML SSO is documented in the readme for Flask SAML SSO.
+
+The following additional configuration entries exist for auth in OS2MO.
```

Comment:
How about roles?

Reply:
What do you mean?

Reply:
I was thinking of group membership, etc., shouldn't we mention that in the documentation?

Reply:
Group membership in terms of being able to create API tokens etc? That's missing, yes. I'd rather add that to the docs for `flask_saml_sso` as that's where the code is.